client/gui/CAnimation.h

```diff
@@ -39,113 +39,113 @@ class IImage
 public:
 	using BorderPallete = std::array<SDL_Color, 3>;
 
-	//draws image on surface "where" at position
-	virtual void draw(SDL_Surface * where, int posX = 0, int posY = 0, const Rect * src = nullptr, ui8 alpha = 255) const=0;
-	virtual void draw(SDL_Surface * where, const SDL_Rect * dest, const SDL_Rect * src, ui8 alpha = 255) const = 0;
+	// draws image on surface "where" at position
+	virtual void draw(SDL_Surface * where, int posX = 0, int posY = 0, const Rect * src = nullptr, ui8 alpha = 255) const = 0;
+	virtual void draw(SDL_Surface * where, const SDL_Rect * dest, const SDL_Rect * src, ui8 alpha = 255) const            = 0;
 
 	virtual std::shared_ptr<IImage> scaleFast(float scale) const = 0;
 
 	virtual void exportBitmap(const boost::filesystem::path & path) const = 0;
 
-	//Change palette to specific player
-	virtual void playerColored(PlayerColor player)=0;
+	// Change palette to specific player
+	virtual void playerColored(PlayerColor player) = 0;
 
-	//set special color for flag
-	virtual void setFlagColor(PlayerColor player)=0;
+	// set special color for flag
+	virtual void setFlagColor(PlayerColor player) = 0;
 
-	//test transparency of specific pixel
+	// test transparency of specific pixel
 	virtual bool isTransparent(const Point & coords) const = 0;
 
 	virtual Point dimensions() const = 0;
-	int width() const;
-	int height() const;
+	int           width() const;
+	int           height() const;

```

Comment:
personally I'm against vertical alignment. Maybe allow it only for enums (if possible).

Reply:
Personally I prefer it but clang-format is just *too* agressive with it.
Problems is, same option is also used for assignments alignment, e.g:
```
	delta             = animDelta;
	fadingMode        = mode;
	fadingCounter     = initialCounter();
	fading            = true;
	shouldFreeSurface = freeSurfaceAtEnd;
```
Where (IMO) vertical alignment is justified.
Even in methods, I don't think anything wrong with aligning int/bool/floats (1-2 extra spaces).
Not 10 spaces like in this example obviously.

In that case I'll bring this up for discussion. My proposition:
- enable this option by default
- whenever clang-format gets too agressive - add empty line between long and short lines. Example: this file in next commit (will push shortly)

Reply:
> Where (IMO) vertical alignment is justified.

there's a simple "counter-example": consider you add one more variable there that becomes the longest one, will you re-format all the other variables just because of that?

> whenever clang-format gets too agressive - add empty line between long and short lines. Example: this file in next commit (will push shortly)

this would work I think

Reply:
If we do have different personal preferences, let's not apply rule and rely on developer. For instance I personally don't use such alignments, because sometimes I use text search like 'var = value' to find where variable gets assigned